src/styles/modules/List.scss

```diff
@@ -0,0 +1,11 @@
+.tasksList {
+    list-style-type: square;
+    .task {
+        margin: 20px 10px;
+        .removeButton {
```

Comment:
names of the classes seems to be not acc to BEM. 

Don't you think, that button could be styled in button part of the css ? 🤔